src/i18n/es.js

```diff
@@ -149,24 +149,26 @@ Encontrarás un Email con los pasos a seguir.`,
     plan_suscribers: 'Suscriptores',
   },
   invoices_list: {
-    account_id_column: `Cuenta Id`,
     amount_column: `Importe`,
     balance_column: `A Saldar`,
     control_panel_account_preferences_url: `${urlControlPanelMain}?section=AccountPreferences`,
     control_panel_billing_information_section: `Información de Facturación`,
     control_panel_billing_information_url: `${urlControlPanel}/AccountPreferences/BillingInformationSettings`,
     control_panel_section: `Panel de Control`,
     creation_date_column: `Fecha Creación`,
+    credit_note_text: `Nota de crédito`,
```

Comment:
The same than before. 
```suggestion
    credit_note_text: `Nota de Crédito`,
```

Reply:
Done